content/uiux/concepts/research-report/research-report.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Research Report'
+Description: 'A research report is a deliverable that summarizes insights from a user research study.'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A **research report** is a deliverable that summarizes insights from a [user research](https://www.codecademy.com/resources/docs/uiux/user-research) study. The report can take various forms, such as a written report or a slide deck. This report should reference the [research plan](https://www.codecademy.com/resources/docs/uiux/research-plan) that was created before the study was conducted.  
+
+A research report should tell a compelling and accurate story about the users. The findings may be shared across an organization, so they should be digestible and actionable. An effective report helps stakeholders make important decisions about the next steps to take in the project or design.  
+
+## Sections 
```

Comment:
[proofreading] Consider renaming the title of this section. It'll help readers gain a better understanding of what is being described. 
```suggestion
## Components of a Research Report
```